server/server.js

```diff
@@ -18,13 +30,31 @@ app.get('/api/teams', (req, res) => {
   res.json(teams);
 });
 app.get('/api/matches', (req, res) => {
-  getDataMatches()
+  getMatches()
     .then(users => res.json(users))
     .catch((err) => {
       console.log(err);
       new Error(err);
     });
 });
+app.post('/api/login', (req, res) => {
+  console.log('im in the server');
+  res.body('success');
+  // getPassword(req.body.username)
+  // .then(hash => bcrypt.compare(req.body.password, hash)
+  // .then(result => {
+  //   if (result === true) {
+  //     res.redirect('/')
+  //   } else {
+  //     res.body('login failed')
+  //   }
+  // })
+  // .catch(err) => {
+  //   console.log(err);
+  //   new Error(err);
+  // })
+});
```

Comment:
As we discussed with @BenNewman90 , res.body() is not function. and that will be handled in the loginhandler branch.